app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs

```diff
@@ -953,6 +953,22 @@
                                     :data-cy "add-contact"}
             (tr [:cooperation :add-application-contact])])]])]))
 
+(def custom-date-formatter
+  (cljs-time.format/formatter "dd.MM.yyyy"))
+
+(defn format-date
+  "Format date to dd.MM.yyyy"
+  [date]
+  (cljs-time.format/unparse custom-date-formatter date))
+
+(defn date-label-component
+  [{value :value label :label}]
+  [:div {:style (merge {:color theme-colors/gray} common-styles/body-2-bold)} label
+   [:div {:style common-styles/body-1-bold}
+    [common/popper-tooltip {:title "Date of application can not be changed"
```

Comment:
English text here? should be translated

Reply:
Yes, waiting for the exact verbiage from design..